modules/ROOT/pages/configure-local-registry.adoc

```diff
@@ -234,6 +234,31 @@ kubectl create secret docker-registry rtf-pull-secret --namespace <rtf-namespace
 . Follow the *Helm* path instructions to download the `values.yml` file so that you can modify its values. For the private registry configuration, update `rtfRegistry` with your local docker server, and update the `pullSecretName` parameter. . 
 . Continue with the *Helm* path instructions and install Runtime Fabric in your Kubernetes cluster.
 
+--
+
+OpenShift::
++
+--
+. <<synch-rtf,Synchronize your container images>> to your local registry.
+. xref:install-helm.adoc#create-a-runtime-fabric-using-runtime-manager[Create a Runtime Fabric using Runtime Manager].
+. In Runtime Manager, select the *Helm* path, and follow the instructions on the screen platform.
+. Obtain your private `docker-server`, `docker-username`, and `docker-password`.
+. Follow the *Helm* path instructions in Runtime Manager to create a secret with your docker server name, username, and password by running the following command in your Unix shell:
++
+[source,copy]
+----
+kubectl create secret docker-registry rtf-pull-secret --namespace <rtf-namespace> --docker-server=<docker-servername> --docker-username=<docker-username --docker-password=<docker-password> --docker-email=<docker-email>
```

Comment:
Let's modify

"kubectl create secret docker registry"

for

"oc create secret docker-registry "

to make it consistent with the installation page.